addons/hr_recruitment/res_config.py

```diff
@@ -52,3 +52,17 @@ def set_default_alias_prefix(self, cr, uid, ids, context=None):
             else:
                 mail_alias.write(cr, uid, alias_id, {'alias_name': record.alias_prefix}, context=context)
         return True
+
+    def get_default_alias_domain(self, cr, uid, ids, context=None):
```

Comment:
Yes this method required to fix alias_domain issue. without this method it's return alias_domain
: {'lang': 'en_US', 'tz': 'Europe/Brussels', 'params': {'action': 185, 'model': 'hr.recruitment.config.settings', '_push_me': False, 'view_type': 'form'}, 'uid': 1}.